### PR TITLE
May2updates

### PR DESCRIPTION
**Pull request by jg10545**

### CODE REVIEW

**Comment by jg10545** [comment]
took out SimCLR batch image viz #34

**Comment by jg10545** [comment]
added function to compute L2 loss for a keras model; integrated with SimCLR and distributed SimCLR #32

**Comment by jg10545** [comment]
feature extraction module can do cosine learning rates now #31

**Comment by jg10545** [comment]
Added SimCLR default augmentation params (#33) and BADGE implementation (#13)
